resources/js/react-app/common/Footer.js

```diff
@@ -42,7 +47,12 @@ export default function Footer() {
 }
 
 const FooterStyled = styled.footer`
-    height: 60px;
+    @media (max-width: 767px) {
+        height: 90px;
+    }
+    @media (min-width: 767px) {
```

Comment:
Diese Media-Query braucht es nicht. Du kannst die height in Zeile 45 lassen. Die Styles sind immer so definiert, dass sie für große Displays okay sind. Nur Abweichungen für kleine Displays werden in Querys spezifiziert. Es ist Best Practice die Styles sozusagen aus einer Richtung kommend zu definieren, entweder Mobile-first oder Desktop-first, und nur die Abweichungen zu spezifizieren. Wir benutzen Desktop-first.

Reply:
Und über und unter Media-Querys bitte immer eine Leerzeile lassen, damit es besser lesbar ist.